README.md

```diff
@@ -1,49 +1,5 @@
 # audio-discrimination-crowdsource
 
```

Comment:
Please explain to me why you deleted my README?

Reply:
I didn't,

Reply:
Then..... look at this diff. It removes all my text. Please review your diffs before pushing